agent/sshd/server.go

```diff
@@ -78,7 +78,6 @@ func NewServer(api client.Client, authData *models.DeviceAuthResponse, privateKe
 			return &sshConn{conn, closeCallback, ctx}
 		},
 	}
-
```

Comment:
Unrelated change, please undo it.

Reply:
@otavio My patch is based on master branch, do you mean I need change the base branch to commit.
If that, what branch should it base on?

Reply:
thanks, removed it.